User.cc

```
@@ -0,0 +1,28 @@
+#include "User.h"
+#include "Portfolio.h"
+#include <memory>
+#include <string>
+#include <iostream>
+using namespace std;
+
+User::User(string username): username{username}, cashBalance{1000} {
+  Portfolio portfolio;
+}
+
+User::~User(){}
+
+void User::addShares(string stockName, int numShares, float currentStockValue){
+  // subtract the stock purchase cost from cash balance
+  cashBalance -= numShares*currentStockValue;
```

Can you throw an exception if the `numShares*currentStockValue > cashBalance`